decidim/decidim
# Add tabs to Attachments, Scopes and Areas

#### :tophat: What? Why?
During the maintainers meeting, we have agreed that we would implement tabbing system so that we can group related elements together. This PR adds the tabbing for the Attachment functionality inside a Participatory Space. 

#### Testing
1. Login as admin and visit any of the participatory processes 
2. Click on the New Attachments buttons 
3. See the new page loading is actually Files section. 
4. Please also observe there is a tab called "Folders" where you can see the attachments collections.
5. Since this works with the menu registry, you will need to restart the server after checkout of the branch. 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
